### ISSUE TITLE
demonstrations of circle folding

### ISSUE THREAD

**Issue by silky** [hahayes/hahayes-projects#11]
apparently there is a thing called "circle folding"

there's a website here - http://wholemovement.com/

http://wholemovement.com/how-to-fold-circles

**Comment by mobeets** [comment]
Hahahah, of man, that guy is hilarious and insane: "So what the cirlce has done in the first folding: it's giving us its origin, which is the sphere, and where it's going, which is the tetrahedron." What does this mean?

**Comment by silky** [comment]
hahahaha; i hadn't watched the video yet. amazing.

**Comment by silky** [comment]
i don't know what it means. but i love it.

**Comment by silky** [comment]
hmm, from the FAQ:

> WHY IS THE CIRCLE SPECIAL MORE THAN OTHER SHAPES? 
> 
> The circle can generate everything that all polygons do, but no polygon can do what the circle 
> does

He could be right I guess. but what shapes am I allowed? Can I build a circle if all I have is an infinite amount of lines? I guess so. Well, it depends how I'm allowed to do it? Discretely I can't, I guess.

And I guess after he gets triangles he can build any shape he wants. So that is cool, infact.

**Comment by silky** [comment]
of course, i suppose if i had a triangle i could just fold the edges constantly until i got a circle. (not physically, of course, as the height would grow exponentially at the folds i think.)

**Comment by mobeets** [comment]
"i don't know what it means. but i love it." <-- this should be the name of the successor to hahayes, if we ever need one. haha
